src/sync.ts

```diff
@@ -94,16 +89,25 @@ export async function exec(cli: meow.Result) {
   }
 
   logger.info(`Executing '${command}' in ${dirs.length} directories.`);
-  for (let i = 0; i < dirs.length; i++) {
-    const dir = dirs[i];
-    logger.info(`[${i + 1}/${dirs.length}] Executing cmd in ${dir}...`);
-    try {
-      await spawn(command[0], command.slice(1), {cwd: dir});
-    } catch (e) {
-      logger.error(dir);
-      logger.error(e);
-    }
-  }
+  let i = 0;
+  const q = new Q({concurrency: 10});
+  const proms = dirs.map(dir => {
+    return q.add(() => {
+      return spawn(command.join(' '), {cwd: dir})
+      .then(r => {
+        i++;
+        logger.info(`[${i}/${dirs.length}] Executing cmd in ${dir}...`);
```

Comment:
This message is misleading since the command is already executed by the time it's printed (it's in `spawn().then()`. Should probably be moved up before `return`.

Reply:
Good catch!Because these are async everywhere, showing this before the command would result in 50 "doing x/y thing" message all at once :) I changed the text to be past tense.